### PR TITLE
Some small improvments

### PR DESCRIPTION
**Pull request by barais**
Two small improvements for my usage. I do not know if it could be useful for others. 

1. I often found comics in which the page name are incorrect in particular regarding the page number. 
1. When I read comic in FitMode width and when I click to next page, I got the next pages but the bottom of the page. I just add a function to automatically scroll to top.

### CODE REVIEW

File: code/comic-book-binder.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class ComicBookBinder extends BookBinder {
 
         if (foundError) {
```
Comment (codedread) [inline code/comic-book-binder.js]: Can you find out why you are getting into this if-branch?  i.e. why is foundError true for your comic book?

(Just log the e values to the console on line 89)

File: code/kthoom.js [hunk 2]
```diff
@@ -692,6 +692,8 @@ class KthoomApp {
 
   showNextPage() {
     const turnedPage = this.bookViewer_.showNextPage();
+    if (this.bookViewer_.getFitMode() === FitMode.Width)
```
Comment (codedread) [inline code/kthoom.js]: Actually this seems like a good change regardless of the fit mode?

However, if you really only want to do this for FitMode.Width, can you pls enclose this if-block with curly braces?

File: code/comic-book-binder.js (hunk 1 above)
Comment (barais) [reply code/comic-book-binder.js]: I can share you for a short period. 

https://drive.google.com/open?id=1rm5RglMmhIkh1HIJa-mQImvQ-bR2w9YI

You will see that the problem is the naming of the file within the archive. As a result, the page order is wrong in kthoom

Regards

Olivier

File: code/kthoom.js (hunk 2 above)
Comment (barais) [reply code/kthoom.js]: Yes of course I will do the update

File: code/comic-book-binder.js (hunk 1 above)
Comment (codedread) [reply code/comic-book-binder.js]: Thanks - I was able to grab the books and take a look.  You are right, the page order is incorrect due to the naming in the file.  Unfortunately, this is a common problem with comic book archive files and readers :-/

I do not get into the foundErr if-block, so I think your change is actually applied to the wrong line (it should be within the sort function).

File: code/kthoom.js [hunk 3]
```diff
@@ -692,6 +692,9 @@ class KthoomApp {
 
   showNextPage() {
     const turnedPage = this.bookViewer_.showNextPage();
+    if (this.bookViewer_.getFitMode() === FitMode.Width) {
```
Comment (codedread) [inline code/kthoom.js]: Can you also add a similar if to the showPrevPage() function?

File: code/comic-book-binder.js [hunk 4]
```diff
@@ -96,7 +96,7 @@ export class ComicBookBinder extends BookBinder {
...
           // TODO: Better error handling.
-          alert('Some pages had errors. See the console for more info.')
+          return a.getPageName().toLowerCase().length > b.getPageName().toLowerCase().length ||  a.getPageName().toLowerCase() > b.getPageName().toLowerCase() ? 1 : -1;
```
Comment (codedread) [inline code/comic-book-binder.js]: We have to be careful not to mess up the sorting for other types of books.  I have a better fix for this - can you revert this file from your pull request?

Comment (barais) [reply code/comic-book-binder.js]: Ok, I just commit with the patch at the right place. I am sorry, I do a small fix for me and I report it for the pull request at the wrong line of code, I just update it in the last commit.

File: code/kthoom.js (hunk 3 above)
Comment (barais) [reply code/kthoom.js]: done in the last commit

**Comment by codedread** [comment]
Thanks for this - I think I have a better fix for your filename sorting, but I'll merge your PR in first and then commit mine on top.

**Pull request merged**